src/fileCreator.ts

```diff
@@ -53,7 +54,10 @@ function replaceName(stringToReplace: string, name: string): string {
 function getTemplateFileNamesAtTemplateDirectory(templateFolderPath: string): string[] {
     const files = fs
         .readdirSync(templateFolderPath)
-        .filter((f) => !fs.statSync(templateFolderPath + "/" + f).isDirectory())
+        .filter((f) => {
+            const folderPath = path.join(templateFolderPath, f);
+            return !fs.statSync(folderPath).isDirectory();
```

Comment:
Something that I noticed while reading up on `fs.statSync` is that the object it returns also has an `isFile()` method.  Not really important in terms of this PR but just wondering if it would make more sense to use that rather than checking whether it is not a directory with the `isDirectory()` method.